src/webapi/Controllers/CandidateController.cs

```diff
@@ -24,17 +24,14 @@ public CandidateController(IODSAPIAuthenticationConfigurationService service, IE
         public async Task<ActionResult<IEnumerable<Candidate>>> GetCandidates()
         {
             var candidates = await _evaluationRepository.GetAllCandidates();
-            if (candidates != null)
-            {
-                return Ok(candidates.Select(c => new Candidate()
+            return candidates != null
```

Comment:
@jleiva-gap The ternary and casting here make this difficult to read, suggest:

`
if (candidates is null) { return Ok(new List<Candidate>()); }
`

`
            return Ok(candidates.Select(c => new Candidate()
            {
                CandidateName = $"{c.FirstName} {c.LastName}",
                PersonId = c.PersonId,
                SourceSystemDescriptor = c.SourceSystemDescriptor
            }).ToList());
`